src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlPathParser/GeneratedStreamGeometryContext.cs

```diff
@@ -14,69 +14,80 @@ internal class GeneratedStreamGeometryContext : StreamGeometryContext
 	{
 		StringBuilder _builder;
 
-		public string Generated { get { return _builder.ToString() + "})"; } }
+		private static readonly string CRLF = Environment.NewLine;
```

Comment:
nit: The name mismatches the value. `Environment.NewLine` may be `\n` (LF) or `\r\n` (CRLF)

Reply:
I know...  I found that `NewLine` was too long.  Anyway that's just to prevent a warning in VisualStudio about mixed line ending mode.

I'm glad you discover it ;-)